src/hooks/useCourtContracts.js

```diff
@@ -625,3 +633,57 @@ export function useTotalANTStakedPolling(timeout = 1000) {
 
   return [totalANTStaked, error]
 }
+
+export function useTotalANTStaked() {
```

Comment:
Just wondering—weren't both of these implemented in the PR for the stats?

In any case, they are both missing the "cancel" pattern :).

Reply:
yes, both of those functions are old ones.. just removed them